CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 ## Version 3.2.7
-* Added parameter in core section of nginx.conf via pillar
+* 3 nginx.conf parameters can be customised via pillar
```

Comment:
I was going to release it as soon as merged - I was hoping you could understand that and help me merge it quickly